app/helpers/encryption_helper.rb

```diff
@@ -0,0 +1,149 @@
+require "csv"
+module EncryptionHelper
+  def assign_secret_key
+    primes = []
+
+    CSV.foreach("primes_test.csv") do |row|
+      primes << row.first.to_i
+    end
+
+    length = primes.length - 1
+
+    key = [primes[rand(length)], primes[rand(length)]]
+
+    while key[0] == key[1]
+      key = [primes[rand(length)], primes[rand(length)]]
+    end
+
+    key
+  end
+
+  def euclidean_algorithm(a, b)
+    euclidean_algorithm = [[a, a/b, b, a%b]]
+    n = 0
+    while euclidean_algorithm[n][3] > 0
+      euclidean_algorithm << [euclidean_algorithm[n][2],
+      euclidean_algorithm[n][2]/euclidean_algorithm[n][3],
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Surrounding space missing for operator '/'.